Cubical/Categories/Presheaf/KanExtension.agda

```diff
@@ -0,0 +1,188 @@
+{-# OPTIONS --cubical --no-import-sorts --safe --experimental-lossy-unification #-}
+
+{-
+  Kan extension of a functor C → D to a functor PreShv C ℓ → PreShv D ℓ.
+  Currently only left Kan extension.
```

Comment:
Aha, my UniMath formalization that I mentioned the other day was of right Kan extensions: https://unimath.github.io/doc/UniMath/4dd5c17/UniMath.CategoryTheory.RightKanExtension.html

That proof followed  MacLane, CWM, X.3 (p. 233) and was also quite long to formalize

Reply:
It seems like I used Theorem 2 (iv) of Chapter IV.1 of MacLane btw

Reply:
I'm only proving a special case of (the left version of) that result, which probably explains why it's shorter. Since I'm in presheaves I can do concrete constructions with HITs instead of only having categorical universal properties.

Reply:
Ah right, good point! Maybe you construction is just an unfolded version of MacLane's? You compute the colimit pointwise in order to define the left Kan extension?

Reply:
Yeah, I would think it has to be more or less the same thing.